src/main/java/org/opendatakit/suitcase/utils/FileUtils.java

```diff
@@ -123,18 +123,18 @@ public static Path getInstancesPath(CloudEndpointInfo cloudEndpointInfo, String
     return getBasePath(cloudEndpointInfo, tableId, savePath).resolve(INSTANCES_PATH);
   }
 
-  public static String getCSVName(CsvConfig config) {
+  public static String getCSVName(CsvConfig config, String tableId) {
     StringBuilder csvNameBuilder = new StringBuilder();
```

Comment:
The tableId should be at the beginning so it is added after line 127.

Reply:
Ohkay